umh-core/pkg/config/manager.go

```
@@ -299,3 +374,63 @@ func (m *FileConfigManagerWithBackoff) IsPermanentFailure() bool {
+func (m *FileConfigManagerWithBackoff) writeConfig(ctx context.Context, config FullConfig) error {
+	// Check if context is already cancelled
+	if ctx.Err() != nil {
+		return ctx.Err()
+	}
+
+	return m.configManager.writeConfig(ctx, config)
+}
+
+// AtomicSetLocation sets the location in the config atomically
+func (m *FileConfigManager) AtomicSetLocation(ctx context.Context, location models.EditInstanceLocationModel) error {
+	m.mutexAtomicUpdate.Lock(ctx)
```

golangci-lint (1.64.8)

Error return value of `m.mutexAtomicUpdate.Lock` is not checked

(errcheck)